stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -542,9 +542,11 @@ public func expectCrashLater() {
   _seenExpectCrash = true
 }
 
-public func expectCrash(executing: () -> Void) {
+public func expectCrash(executing: () -> Void) -> Never {
   expectCrashLater()
   executing()
+  expectUnreachable()
+  fatalError()
```

Comment:
This one is unnecessary ;-) `expectUnreachable` already terminates execution.

Reply:
It *would* be unnecessary if `expectUnreachable` returned `Never`…